Parking_Zone.Test/ModelValidationtests/ParkingSlots/CreateVMTest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Parking_Zone.Domain.Enums;
+using Parking_Zone.MVC.Models.ParkingSlotVMs;
+using System.ComponentModel.DataAnnotations;
+using Xunit;
+
+namespace Parking_Zone.Test.ModelValidationtests.ParkingSlots;
+
+public class CreateVMTest
+{
+    public static IEnumerable<object[]> TestData =>
+      new List<object[]>
+      {
+                new object[] { 5, new ParkingSlotCategory(), false, 20, true },
```

Comment:
give here value just one of the categories like ParkingSlotCategorty.VIP or .Business do not just call the class